### ISSUE TITLE
RNA-seq - featureCounts on whole gene body (including introns) missing

### ISSUE THREAD

**Issue by mirax87** [maxplanck-ie/snakepipes#508]
Currently, featureCounts takes the genes.filtered.gtf (output from filterGTF.snakefile) that contains transcripts, exons and further features, but not the feature type _gene_, which is required to quantify the full gene with introns. 

One way to achieve that is to leave the gtf filtering for featureCounts to feautreCounts itself, instread of feeding it with the filtered gtf.

My application here is to quantify GRO-seq on whole gene bodies, which captures also nascent transcripts (a more straight-forward application probably would be gene quantification on total RNA-seq).

**Comment by katsikora** [comment]
Hi Michael,

This is the default and intended so. 
Your request can be handled by either adding another analysis mode to RNA-seq that would switch to counting on 'gene', or, if this is the only modification you need for GRO-seq, by adding a param to filterGTF/featureCounts that specifies the feature type.

**Comment by katsikora** [comment]
Actually you could pass feature type to feature counts via --featureCountsOptions ' -t gene ' as a commandline argument.

**Comment by mirax87** [comment]
Actually, there is no feature type "gene" in the gtf, that is used by featureCounts, i.e. the gtf "Annotation/genes.filtered.gtf". 

I checked in the following way:

Running `featureCounts -t gene -a  Annotation/genes.filtered.gtf *.bam` fails and complains about the missing feature type "gene"

Looking at the `Annotation/genes.filtered.gtf`, there is no gene feature present. 
```
> cut -f 3 Annotation/genes.filtered.gtf  | sort | uniq
3UTR
5UTR
CDS
exon
start_codon
stop_codon
transcript
```

**Comment by dpryan79** [comment]
Thankfully this will get fixed once #510 is merged, since then we're not doing this weird GTF->BED->GTF conversion any more.

**Comment by dpryan79** [comment]
I've confirmed that you can now correctly use `--featureCountsOptions " -t gene"` in the develop branch.
